js/layout-javascript/agenda-code.js

```diff
@@ -903,6 +904,14 @@ DynamicList.prototype.attachObservers = function() {
       $parent.find('.btn-bookmarked').focus();
 
       record.bookmarkButton.like();
+    })
+    .on('click', '.multiple-images-item', function() {
```

Comment:
Add accessibility enhancements to make sure users can open the images with keyboard.